makefile

```diff
@@ -7,23 +7,47 @@ SFILES      = *.s
 SRCDIR      = src
 OBJDIR      = obj
 BINDIR      = bin
-VC_VERSIONS = D43J NACJ NACE NARJ NARE
+VC_VERSIONS = PZLJ PZLE
 NAME        = homeboy
 RESDESC     = res.json
 
+# either 'OOT' or 'MM'
+GAME ?= OOT
+
+ifeq ($(GAME),)
+$(error "GAME unset.")
+else ifneq ($(GAME),OOT)
+ifneq ($(GAME),MM)
+$(error "Wrong value for GAME.")
+endif
+endif
+
 ADDRESS             = 0x817F8000
-ALL_CFLAGS          = -c -Iinclude -mcpu=750 -meabi -mhard-float -G 0 -O3 -ffunction-sections -fdata-sections $(CFLAGS)
-ALL_CPPFLAGS        = $(CPPFLAGS)
+ALL_CFLAGS          = -c -Iinclude -mcpu=750 -meabi -mhard-float -G 0 -O3 -ffunction-sections -fdata-sections $(CFLAGS) -DGAME=$(GAME)
+ALL_CPPFLAGS        = $(CPPFLAGS) -DGAME=$(GAME)
```

Comment:
I think this passes `-DGAME=OOT` twice in the command line? Probably `ALL_CPPFLAGS` only is most appropriate